### ISSUE TITLE
Error in result of call to scipy.stats.ttest_ind_from_stats

### ISSUE THREAD

**Issue by brian-rose** [mikemorris12/UTCDW_Guidebook#4]
As part of my [review of the JOSE submission](https://github.com/openjournals/jose-reviews/issues/243) I'm going through all the notebooks.

In [Section 6.4.2 Exploratory Analysis](https://utcdw.physics.utoronto.ca/UTCDW_Guidebook/Chapter6/section6.4_station_1D_example_nearterm.html#exploratory-analysis) I'm getting a failure after the call to `scipy.stats.ttest_ind_from_stats`. The code in the notebook is 
```
# perform two_sample t-test to see if future temperatures are higher than past
pvals = []
for scenario in tas_ens_future_raw_mean.scenario.values:
    tstat, pval = stats.ttest_ind_from_stats(tas_ens_future_raw_mean.sel(scenario = scenario), 
                                            tas_ens_future_raw_stdev.sel(scenario = scenario), 
                                            neff_future_raw.sel(scenario = scenario), 
                                            tas_ens_hist_raw_mean,
                                            tas_ens_hist_raw_stdev,
                                            neff_hist_raw, 
                                            equal_var = False,
                                            alternative = 'greater') 
    pvals.append(pval.values)

    print('p-values for mean change of raw tas')
```

Taking `pval.values` throws the error
```
AttributeError: 'numpy.float64' object has no attribute 'values'
```

This seems consistent with the [scipy docs](https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.ttest_ind_from_stats.html) which state that the return values `tstat`, `pval` should be float or array.

**Comment by brian-rose** [comment]
I changed the line
```
    pvals.append(pval.values)
```
to simply
```
    pvals.append(pval)
```
and the cell seemed to execute normally.

**Comment by mikemorris12** [comment]
Thanks for pointing out this bug. I didn't encounter it since when I ran it, the function `scipy.stats.ttest_ind_from_stats` returned the p-values in an `xr.DataArray`, from which I extracted the values in a `numpy` array using the `.values` accessor. Now it seems to be converting the output to a `numpy` array, so I've incorporated the suggested change in commit [ba5041b](https://github.com/mikemorris12/UTCDW_Guidebook/commit/ba5041b86f8eb3db55dcf53aedf11e5019dd0340). Thanks!
